test/cell_test.rb

```diff
@@ -0,0 +1,67 @@
+require "minitest/autorun"
+require "minitest/pride"
+require "./lib/ship"
+require './lib/cell'
+
+class Test < Minitest::Test
+  def test_it_exists
+    cell = Cell.new("B4")
+
+    assert_instance_of Cell, cell
+  end
+
+  def test_attributes
+    cell = Cell.new("B4")
+
+    assert_equal "B4", cell.coordinate
+  end
+
+  def test_ship_in_cell
+    cell = Cell.new("B4")
+
+    assert_equal nil, cell.ship
```

Comment:
Suggest changing this to 'assert_nil' to avoid error in mintiest

Reply:
Adjusted line 22 to 'assert_nil'